<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefControlsDlg</class>
 <widget class="QWidget" name="DlgPrefControlsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxRateRamp">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Makes temporary speed buttons change the pitch gradually, instead of all at once.</string>
     </property>
     <property name="title">
      <string>Ramping Pitchbend</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pitchbend sensitivity</string>
        </property>
        <property name="buddy">
         <cstring>SliderRateRampSensitivity</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="SliderRateRampSensitivity">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>2500</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="SpinBoxRateRampSensitivity">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Value in milliseconds</string>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>2500</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Temporary Speed Adjustment Buttons</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="TextLabel6_2_3_2_3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font/>
            </property>
            <property name="text">
             <string>Left click</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>spinBoxTempRateLeft</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="TextLabel6_2_3_2_2_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font/>
            </property>
            <property name="text">
             <string>Right click</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>spinBoxTempRateRight</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxTempRateLeft">
            <property name="toolTip">
             <string>Temporary rate change when left-clicking</string>
            </property>
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>4.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxTempRateRight">
            <property name="toolTip">
             <string>Temporary rate change when right-clicking</string>
            </property>
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Permanent Speed Adjustment Buttons</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="TextLabel6_2_3_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font/>
            </property>
            <property name="text">
             <string>Left click</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>spinBoxPermRateLeft</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="TextLabel6_2_3_2_2">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font/>
            </property>
            <property name="text">
             <string>Right click</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>spinBoxPermRateRight</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxPermRateLeft">
            <property name="toolTip">
             <string>Permanent rate change when left-clicking</string>
            </property>
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxPermRateRight">
            <property name="toolTip">
             <string>Permanent rate change when right-clicking</string>
            </property>
            <property name="accelerated">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.050000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="GridLayout1">
     <item row="10" column="0">
      <widget class="QLabel" name="labelResetSpeedAndPitch">
       <property name="text">
        <string>Reset speed and pitch</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelCueMode">
       <property name="text">
        <string>Cue mode</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
       <property name="buddy">
        <cstring>ComboBoxCueDefault</cstring>
       </property>
      </widget>
     </item>
     <item row="13" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxRateDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font/>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxPosition">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font/>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="warningLabel">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelFullscreenOption">
       <property name="text">
        <string>Start in full screen mode</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelPositionDisplay">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Position display</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>ComboBoxPosition</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lableSkin_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Skin</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>ComboBoxSkinconf</cstring>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="lableSpeedSliderrange">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Speed slider range</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>ComboBoxRateRange</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxTooltips">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="toolTip">
        <string>A small pop-up box displays a brief description of a control's purpose when the cursor rests on the control.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="2" colspan="2">
      <widget class="QComboBox" name="ComboBoxSchemeconf">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="toolTip">
        <string>Select from different color schemes of a skin if available.</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxRateRange">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="toolTip">
        <string>Adjusts the range of the speed (Vinyl &quot;Pitch&quot;) slider.</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="lableSpeedSliderDirection">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Speed slider direction</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>ComboBoxRateDir</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="2">
      <widget class="QLabel" name="lableColorScheme">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Color scheme</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>ComboBoxSchemeconf</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxSkinconf">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelTooltips">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="text">
        <string>Tool tips</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>ComboBoxTooltips</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxStartInFullscreen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="14" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxKeylockMode"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="labelKeylockMode">
       <property name="text">
        <string>Keylock mode</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxSeekToCue">
       <property name="toolTip">
        <string>Automatically seeks to the first saved cue point on track load. If none exists, seeks to the beginning of the track.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="textLabelLocale">
       <property name="text">
        <string>Locale</string>
       </property>
       <property name="buddy">
        <cstring>ComboBoxLocale</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxCueDefault">
       <property name="toolTip">
        <string>Mixxx mode:
- Cue button while pause at cue point = preview
- Cue button while pause not at cue point = set cue point
- Cue button while playing = pause at cue point
Mixxx mode (no blinking):
- Same as Mixxx mode but with no blinking indicators
Pioneer mode:
- Same as Mixxx mode with a flashing play button
Denon mode:
- Cue button at cue point = preview
- Cue button not at cue point = pause at cue point
- Play = set cue point
Numark mode:
- Same as Denon mode, but without a flashing play button
</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="textLabelCueRecall">
       <property name="text">
        <string>Seek to cue</string>
       </property>
       <property name="buddy">
        <cstring>ComboBoxSeekToCue</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxLocale">
       <property name="toolTip">
        <string>Locales determine country and language specific settings.</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="textLabelAllowTrackLoadToPlayingDeck">
       <property name="text">
        <string>Playing track protection</string>
       </property>
       <property name="buddy">
        <cstring>ComboBoxAllowTrackLoadToPlayingDeck</cstring>
       </property>
      </widget>
     </item>
     <item row="11" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxAllowTrackLoadToPlayingDeck"/>
     </item>
     <item row="10" column="1" colspan="2">
      <widget class="QComboBox" name="ComboBoxResetSpeedAndPitch"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ComboBoxSkinconf</tabstop>
  <tabstop>ComboBoxSchemeconf</tabstop>
  <tabstop>ComboBoxStartInFullscreen</tabstop>
  <tabstop>ComboBoxTooltips</tabstop>
  <tabstop>ComboBoxPosition</tabstop>
  <tabstop>ComboBoxRateRange</tabstop>
  <tabstop>ComboBoxRateDir</tabstop>
  <tabstop>spinBoxPermRateLeft</tabstop>
  <tabstop>spinBoxPermRateRight</tabstop>
  <tabstop>spinBoxTempRateLeft</tabstop>
  <tabstop>spinBoxTempRateRight</tabstop>
  <tabstop>groupBoxRateRamp</tabstop>
  <tabstop>SliderRateRampSensitivity</tabstop>
  <tabstop>SpinBoxRateRampSensitivity</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>SliderRateRampSensitivity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SpinBoxRateRampSensitivity</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SpinBoxRateRampSensitivity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SliderRateRampSensitivity</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
